libnetwork/endpoint_cnt.go

```diff
@@ -105,15 +105,15 @@ func (ec *endpointCnt) EndpointCnt() uint64 {
 }
 
 func (ec *endpointCnt) updateStore() error {
-	store := ec.n.getController().getStore()
```

Comment:
This also removes the `getController()` part, which _does_ appear to have synchronisation; is it safe to skip it here?

At least at a quick glance, it looks like there's potential paths that mutate that field, so maybe we still need it?

Reply:
It's safe to remove this call -- there's no way `n.ctrler` can be mutated once the network is loaded from the datastore. But I've undone this change and will do that in a separate commit / PR.